### PR TITLE
#169724219: Implement add pull request template

### PR DESCRIPTION
**Pull request by missvicki**
# What does this PR do?
- Adds a Pull request template

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### PR Description
+
+What PR does?
```
Comment (kcharles52) [inline PULL_REQUEST_TEMPLATE.md]:
```suggestion
### What PR does?
```

**Review by kcharles52 — CHANGES_REQUESTED** [review]
Please add the missed #

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+### What PR does?
```
Comment (katunold) [inline PULL_REQUEST_TEMPLATE.md]: **What this PR does?** instead of **WHat PR does?**

**Review by katunold — APPROVED** [review]
Well done, just look at the small feedback I have left behind for you. This looks good for starters

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (missvicki) [reply PULL_REQUEST_TEMPLATE.md]: @katunold this has been corrected

**Pull request merged**